definitions/procedures/foreman_login_session.rb

```diff
@@ -0,0 +1,19 @@
+class Procedures::ForemanLoginSession < ForemanMaintain::Procedure
```

Comment:
@patilsuraj767 I'm sorry for missing that earlier but the name and description of the remediation procedure should provide more precise information about what we are going to do. As a customer I may think that 'Clean login sessions' may do more than I want. Would 'Reset login session id sequence' sound appropriate? Also from class name the label of the procedure is deduced so It may be handy to have a verb in it for better orientation (ResetLoginSessionIds perhaps?). Also the file name would need some update.

Reply:
@mbacovsky Thanks for suggesting that. Should I give the file name as reset_login_session_id_sequence.rb and the class name as ResetLoginSessionIds? Also, Should I change the description with the same?